keylime/da/record.py

```diff
@@ -0,0 +1,321 @@
+import base64
+import errno
+import importlib
+import os
+import pickle
+import tempfile
+from datetime import datetime
+from urllib.parse import urlparse
+
+from keylime import config, crypto, json, keylime_logging, web_util
+from keylime.fs_util import ch_dir
+
+logger = keylime_logging.init_logging("persistent_store")
+
+
+class BaseRecordManagement:
```

Comment:
Because this is used to implement other backbends, can you please add type hints to all the public functions and a short doc string on how to use them?

Reply:
done

Reply:
done